app/src/app/api/v0/city/[city]/route.ts

```diff
@@ -19,9 +19,10 @@ export const GET = apiHandler(
         {
           model: db.models.User,
           as: "users",
-          // where: {
-          //   userId: session.user.id,
-          // },
+        },
+        {
+          model: db.models.Population,
+          as: "populations",
```

Comment:
Perhaps keep the comment here.
It was commented out to prevent errors due to access control during demos etc., but we should look into adding the access control back in once there is a way to invite others to your inventory (or that multiple people can have an inventory for the same city).